node/src/Pos/Util/Servant.hs

```diff
@@ -442,15 +441,14 @@ applyServantLogging configP methodP paramsInfo showResponse action = do
                         (colorizeDull Red "Unexecuted request due to error") e
             Right paramLogs ->
                 performLogging $ paramLogs <> msg
-    reportResponse timer resp = do
+    reportResponse timer = do
         durationText <- timer
         logWithParamInfo $
-            sformat ("  "%stext%" "%stext%" "%stext%" "%stext%" "%stext)
+            sformat ("  "%stext%" "%stext%" "%stext%" "%stext)
```

Comment:
Why do we cut out the response?

Reply:
we planned to write this message to public logs, however, a response may contain sensitive info

Reply:
@dcoutts ping